Resources/Locale/en-US/_CM14/squads.ftl

```diff
@@ -18,6 +18,10 @@ department-Marine-description = They form the main fighting body of the Almayer
 department-MarineEngineering = Engineering
 department-MarineEngineering-description = Your job is to maintain the ship's engine and keep everything running.
 
+# medical
+department-MarineMedical = Medical
+department-MarineMedical-description = Treat all patients that come into medbay. Transfer injured marines from the hanger to medbay.
+
```

Comment:
I only just noticed but departments probably shouldnt be in the marine squads ftl lol.
We can move them later for clarity but it's fine for now.